### PR TITLE
Patch for atomic renderer snapshot tests

### PR DESCRIPTION
**Pull request by talagluck**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- This address failures that are currently occurring in our Azure pipelines due to the version of vega-lite in the cdn not matching existing snapshots tests. This is currently a hard-coded patch, but in the future, we need to understand why we are not getting consistent versions from the cdn.
- We could also pin a more specific version of vega-lite, though it doesn't make sense to do this in order to pass a snapshot test.

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: tests/expectations/test_expectation_atomic_renderers.py [hunk 1]
```diff
@@ -178,6 +178,14 @@ def test_atomic_prescriptive_summary_expect_column_kl_divergence_to_be_less_than
 
     res = rendered_content.to_json_dict()
     pprint(res)
+
+    # replace version of vega-lite in res to match snapshot test
+    current_graph_string = res["value"]["graph"]
+    start_index = current_graph_string.index("vega-lite/") + len("vega-lite/")
+    end_index = current_graph_string.index(".json")
+    modified_graph_string = current_graph_string[:start_index] + "v4.8.1" + current_graph_string[end_index:]
+    res["value"]["graph"] = modified_graph_string
```
Comment (cdkini) [inline tests/expectations/test_expectation_atomic_renderers.py]:
```python
current_graph_string = res["value"]["graph"]
modified_graph_string = re.sub(r"v\d*\.\d*\.\d*", "v4.8.1", current_graph_string)
res["value"]["graph"] = modified_graph_string

# OR 

res["value"]["graph"] = re.sub(r"v\d*\.\d*\.\d*", "v4.8.1",  res["value"]["graph"])
```

Comment (cdkini) [reply tests/expectations/test_expectation_atomic_renderers.py]: @talagluck I found a possibly solution using regex if that helps clean things up!

File: tests/expectations/test_expectation_atomic_renderers.py [hunk 2]
```diff
@@ -178,6 +179,10 @@ def test_atomic_prescriptive_summary_expect_column_kl_divergence_to_be_less_than
 
     res = rendered_content.to_json_dict()
     pprint(res)
+
+    # replace version of vega-lite in res to match snapshot test
+    res["value"]["graph"] = re.sub(r"v\d*\.\d*\.\d*", "v4.8.1", res["value"]["graph"])
+
```
Comment (cdkini) [inline tests/expectations/test_expectation_atomic_renderers.py:185]: would you mind adding that comment to the other substitutions 🙇🏽

**Review by roblim — APPROVED** [review]
LGTM, pending passing checks.

**Pull request merged**